src/code.rs

```diff
@@ -193,13 +193,19 @@ impl<'a> Struct<'a> {
 
     /// Assemble the `derive` attribute for the struct
     fn attr_derive(&self) -> String {
-        let mut derives_vec = match &self.config.options.additional_derives {
-            Some(v) => {
-                let mut _derives_vec = Vec::<&str>::with_capacity(10 + v.len());
-                _derives_vec.extend(v.iter().map(|s| -> &str { s.as_ref() }));
-                _derives_vec
-            }
-            None => Vec::with_capacity(10),
+        let mut derives_vec = if self.config.options.additional_derives.is_empty() {
+            Vec::with_capacity(10)
+        } else {
+            let mut _derives_vec =
+                Vec::<&str>::with_capacity(10 + self.config.options.additional_derives.len());
+            _derives_vec.extend(
+                self.config
+                    .options
+                    .additional_derives
+                    .iter()
+                    .map(|s| -> &str { s.as_ref() }),
+            );
+            _derives_vec
```

Comment:
This if and duplicated vec construction could be removed by just using the else case entirely as there is now no reason why `self.config.options.additional_derives` is not just a `vec`.

Reply:
Fair, it's leftover from when it was an `Option`